jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfig.java

```diff
@@ -116,7 +116,9 @@ DockerCredentialHelper getCredentialHelperFor(String registry) {
   }
 
   /**
-   * Registry alias matches in the following order:
+   * Gets alias matches for a registry.
```

Comment:
These are not aliases.
```suggestion
   * Gets registry matchers for a registry.
```